core/arch/arch.h

```diff
@@ -567,6 +567,28 @@ cleanup_after_clean_call(dcontext_t *dcontext, clean_call_info_t *cci,
 void convert_to_near_rel(dcontext_t *dcontext, instr_t *instr);
 instr_t *convert_to_near_rel_meta(dcontext_t *dcontext, instrlist_t *ilist,
                                   instr_t *instr);
+#ifdef AARCH64
+void
+insert_save_registers(dcontext_t *dcontext, instrlist_t *ilist, instr_t *instr,
```

Comment:
This is only used in mangle.c -- why not just make it static there?  Why declare here?

I think the 3 below are that way as well.

Reply:
Done. The *_inline_* versions are used by in clean_call_opt.c